src/cli.js

```diff
@@ -91,10 +97,44 @@ export default function run() {
     type: 'list',
     message: 'Choose your Containerization Tool:',
     choices: containerizationChoices.choices
-  }
-  ];
+  }];
+
+  return inquirer.prompt(questions);
+}
+
+/**
+ * Gets the user's github credentials, logs them in, then safely stores their credentials somewhere.
+ */
+function getGithubCredentials() {
+  const questions = [{
+    name: 'githubUsername',
+    type: 'input',
+    message: 'GitHub Username:',
+  }, {
+    name: 'githubPassword',
+    type: 'password',
+    message: 'GitHub Password'
+  }];
 
   inquirer.prompt(questions).then((answers) => {
-    writeFiles(answers);
+    // TODO log the user into github and safely store their credentials/tokens somewhere
+    console.log(answers);
   });
 }
+
+
+/**
+ * The main execution function for hammer-cli.
+ */
+export default function run() {
+  console.log(chalk.yellow(figlet.textSync('hammer-io', { horizontalLayout: 'full' })));
+
+
+  getConfigs()
+    .then((answers) => {
+      initProject(answers)
+    })
+    .then(() => {
+      getGithubCredentials();
+    });
```

Comment:
All of the .then statements can be changed to async await. Example -
`const answers = await getConfigs()`
`await initProject(answers)`
`await getGithubCredentials`
We can essentially eliminate nasty callbacks and callback hell this way.  This makes the code much more readable. This should be done throughout the project.